services-camel/src/main/java/edu/unc/lib/dl/services/camel/replication/ReplicationProcessor.java

```diff
@@ -47,6 +47,7 @@
  * @author lfarrell
  *
  */
+@Deprecated
```

Comment:
How is replication going to work now?

Reply:
Ah right, longleaf :)

Reply:
Yeah, going to have to add that integration now that the fun of doing it for hyc is complete.